variables.tf

```diff
@@ -109,6 +109,16 @@ variable "skip_scc_wp_auth_policy" {
   description = "Set to true to skip the creation of an IAM authorization policy that permits the SCC instance created by this solution read access to the workload protection instance. Only used if `attach_wp_to_scc_instance` is set to true."
 }
 
+variable "custom_integrations" {
+  type = list(object({
+    attributes       = optional(map(string), {})
+    provider_name    = string
+    integration_name = string
+  }))
+  description = "A list of custom provider integrations to associate with the SCC instance."
+  default     = []
```

Comment:
you may want a `nullable = false` here so that if null is specificaly passed it doesn't error on your for_each.